docs/core/tools/dotnet-new-sdk-templates.md

```diff
@@ -169,6 +170,7 @@ Each template may have additional options available. The core templates have the
 
```

Comment:
Is the default version of net5.0 on line 115 still correct?

Reply:
@vlada-shubina What is the default value for --framework for the wpf templates in .NET 6 SDK?

Reply:
The default value for wpf template is net6.0